Source/WebKit/UIProcess/ProcessThrottler.cpp

```diff
@@ -316,8 +320,10 @@ void ProcessThrottler::setAllowsActivities(bool allow)
 
 void ProcessThrottler::setShouldTakeSuspendedAssertion(bool shouldTakeSuspendedAssertion)
 {
-    m_shouldTakeSuspendedAssertion = shouldTakeSuspendedAssertion;
+#if PLATFORM(COCOA)
```

Comment:
I think ideally only the linkedOnOrAfterSDKWithBehavior check would be protected by the `#if` check. There is no reason the rest of the logic cannot remain cross-platform I think?